cpu/cc2538/periph/timer.c

```diff
@@ -80,6 +115,36 @@ static inline void _irq_enable(tim_t tim)
     }
 }
 
+static inline void _timer_clock_enable(tim_t tim)
+{
+    DEBUG("%s\n", __FUNCTION__);
+
+    SYS_CTRL->RCGCGPT |= (1UL << tim);
+    SYS_CTRL->SCGCGPT |= (1UL << tim);
+    SYS_CTRL->DCGCGPT |= (1UL << tim);
```

Comment:
What are those three registers?

Reply:
These registers define if GPTx clocks are gated in `active (run)`, `sleep`, and `PM0` respectively.

Reply:
Please add a comment.